src/components/breadcrumb/breadcrumb.stories.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { Breadcrumb } from './breadcrumb';
+import Header from '../header/header';
+import { GlobalStyle } from '../layout/global-style';
+import AngularIMG from '../../../blog-posts/images/angular.png';
+
+export default {
+  component: Breadcrumb,
+  title: 'Components/Breadcrumb',
+  decorators: [
+    (Story) => (
+      <>
+        <GlobalStyle $lightTheme={true} />
+        <Story />
+      </>
+    ),
+  ],
```

Comment:
Is this needed? The general storybook setup already pretty much adds the same decorator. Do we need that explicit `$lightTheme` input? We might as well add it to the global decorator in `.storybook/preview.js` since we are currently working only on the light theme.